Dockerfile

```diff
@@ -2,10 +2,15 @@
 ARG GIT_COMMIT=unknown
 ARG GIT_TAG=unknown
 ARG GIT_TREE_STATE=unknown
+ARG BUILDER_IMAGE=golang:1.21-alpine3.18
+ARG UI_IMAGE=node:20-alpine
+ARG DISTROLESS_IMAGE=gcr.io/distroless/static
 
-FROM golang:1.21-alpine3.18 as builder
+FROM $BUILDER_IMAGE as builder
 
-RUN apk update && apk add --no-cache \
+RUN apk update && \
+    (command -v go >/dev/null 2>&1 || apk add --no-cache go) && \
```

Comment:
we should add versions if we want this to match the regular alpine images.

this also feels very hacky to have different base images, but it sounds like there's no pre-built Alpine image with Go and Node for RISC-V?